src/Explorer/Controls/ThroughputInput/ThroughputInput.tsx

```diff
@@ -225,98 +82,148 @@ export class ThroughputInput extends React.Component<ThroughputInputProps, Throu
     const multimasterEnabled: boolean = databaseAccount.properties.enableMultipleWriteLocations;
 
     return PricingUtils.getEstimatedSpendAcknowledgeString(
-      this.state.throughput,
+      throughput,
       userContext.portalEnv,
       numberOfRegions,
       multimasterEnabled,
-      this.state.isAutoscaleSelected
+      isAutoscaleSelected
     );
-  }
+  };
 
-  private onAutoscaleRadioBtnChange(event: React.ChangeEvent<HTMLInputElement>): void {
-    if (event.target.checked && !this.state.isAutoscaleSelected) {
-      this.setState({ isAutoscaleSelected: true, throughput: AutoPilotUtils.minAutoPilotThroughput });
-      this.props.setIsAutoscale(true);
+  const handleOnChangeMode = (ev: React.FormEvent<HTMLInputElement>, option: IChoiceGroupOption): void => {
+    if (option.key === "true") {
+      setThroughputValue(AutoPilotUtils.minAutoPilotThroughput);
+      setIsAutoscale(true);
+    } else {
+      setThroughputValue(SharedConstants.CollectionCreation.DefaultCollectionRUs400);
+      setIsAutoscale(false);
     }
-  }
+  };
 
-  private onManualRadioBtnChange(event: React.ChangeEvent<HTMLInputElement>): void {
-    if (event.target.checked && this.state.isAutoscaleSelected) {
-      this.setState({
-        isAutoscaleSelected: false,
-        throughput: SharedConstants.CollectionCreation.DefaultCollectionRUs400,
-      });
-      this.props.setIsAutoscale(false);
-      this.props.setThroughputValue(SharedConstants.CollectionCreation.DefaultCollectionRUs400);
-    }
-  }
-}
-
-interface CostEstimateTextProps {
-  requestUnits: number;
-  isAutoscale: boolean;
-}
-
-const CostEstimateText: React.FunctionComponent<CostEstimateTextProps> = (props: CostEstimateTextProps) => {
-  const { requestUnits, isAutoscale } = props;
-  const { databaseAccount } = userContext;
-  if (!databaseAccount?.properties) {
-    return <></>;
-  }
+  const optionList: IChoiceGroupOption[] = [
+    { key: "true", text: "Autoscale" },
+    { key: "false", text: "Manual" },
+  ];
+  return (
+    <div className="throughputInputContainer throughputInputSpacing">
+      <Stack horizontal>
+        <span className="mandatoryStar">*&nbsp;</span>
+        <Text variant="small" style={{ lineHeight: "20px", fontWeight: 600 }}>
+          {getThroughputLabelText()}
+        </Text>
+        <InfoTooltip>{PricingUtils.getRuToolTipText()}</InfoTooltip>
+      </Stack>
+
+      <Stack horizontal verticalAlign="center">
+        <ChoiceGroup
+          selectedKey={"" + isAutoscaleSelected}
+          options={optionList}
+          onChange={handleOnChangeMode}
+          aria-label="mode"
+        />
+      </Stack>
+
+      {isAutoscaleSelected && (
+        <Stack className="throughputInputSpacing">
+          <Text variant="small" data-testid="ruDescription">
```

Comment:
Same as the previous comments. Please try to use `aria-label` for testing purposes if possible.